src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Publisher.kt

```diff
@@ -56,4 +52,15 @@ class Publisher(private val topicName: String) {
             publisher?.shutdown()
         }
     }
+
+    companion object {
+        fun createMessage(message: String): PubsubMessage? {
```

Comment:
Why do you return null nullable type? You have no chance to return null from this code.

Reply:
Done